Viktor's readme.tex

```diff
@@ -0,0 +1,74 @@
+\documentclass[a4paper]{article}
```

Comment:
This file should be named something more descriptive like `project-description.tex` or `proposal.tex`.